app/views/user/attendances/show.html.erb

```diff
@@ -1,8 +1,11 @@
 <h1><%= link_to @module.name, turing_module_path(@module) %></h1>
+<%= link_to "Delete Attendance", attendance_path(@attendance), class: 's-button danger', data: { turbo: true, turbo_method: :delete, turbo_confirm: "WARNING!! \n\n Are you sure you want to delete this attendance?" } %>
+<%= render partial: "shared/attendance_details", locals: {attendance: @attendance, updatable: true} %>
```

Comment:
This is the line that's causing CI to fail. I removed this partial in the latest version so you can fix by deleting this line

Reply:
deleted line 3 containing the partial